src/Cache/CacheAdapter.php

```diff
@@ -0,0 +1,262 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Skautis\Nette\Cache;
+
+use DateInterval;
+use Exception;
+use Nette\Caching\Cache;
+use Psr\SimpleCache\CacheInterface;
+use Tracy\Debugger;
+use Traversable;
+
+
+/**
+ * Nette cache adapter for Skautis library
+ */
+class CacheAdapter
+  implements
+  CacheInterface
+{
+
+  /**
+   * @var Cache
+   */
+  private $cache;
+
+  /**
+   * @var int|null
+   */
+  private $defaultTTL;
+
+  public function __construct(
+    Cache $cache,
+    $defaultTTLSeconds = null
+  ) {
+    $this->cache = $cache;
+    $this->defaultTTL = $defaultTTLSeconds;
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function get(
+    $key,
+    $default = null
+  ) {
+    $this->assertValidKey($key);
+
+    try {
+      $value = $this->cache->load(
+        $key,
+        static function () use
```

Comment:
Je toto chování žádoucí? Tohle totiž tu výchozí hodnotu uloží do cache, což by nastat nemělo. PSR to tak nepředepisuje, Symfony/Cache (, které má asi nejpoužívanější implementaci) [tak také nefunguje](https://github.com/symfony/cache/blob/634fdd75503607d5584632f17288405f24788eae/Psr16Cache.php#L83) ale hlavně je to IMO neočekávané chování.

Správně je IMO:
```php
try {
  return $this->cache->load($key) ?? $default;
} catch(Throwable $e) {
  throw new CacheException("Failed to load key '$key''", $exception);
}
```